cognite_toolkit/_cdf_tk/resource_classes/transformations.py

```diff
@@ -0,0 +1,151 @@
+from typing import Literal
+
+from pydantic import Field, ValidationError, model_validator
+
+from .base import ToolkitResource
+
+
+class DataModelInfo(ToolkitResource):
```

Comment:
Lets use `BaseModelResource` for all nested part of a Toolkit resource. `DataModelInfo` is not a Toolikit resource.
```suggestion
class DataModelInfo(BaseModelResource):
```